samples/40.timex-resolution/src/main/java/com/microsoft/bot/sample/timex/resolution/Ambiguity.java

```diff
@@ -21,10 +21,14 @@
  * A TIMEX expression can include a date and time parts. So ambiguity of date can be combined with multiple results.
  * Code that deals with TIMEX expressions is frequently dealing with sets of TIMEX expressions.
  */
-public class Ambiguity {
-
+public final class Ambiguity {
+    private Ambiguity() {
+    }
+    
+    /**
+     * Run the recognizer.
+     */
```

Comment:
This is not okay, this is not the correct javadoc for the method, you are moving the comments 🤔 . I noticed that the rest of the languages in samples doesn't contain the docs, so we should create one